app/src/main/res/layout/default_group_chip.xml

```diff
@@ -13,5 +13,5 @@
     tools:layout_width="wrap_content"
     app:chipMinTouchTargetSize="0dp"
     android:layout_margin="4dp"
-    android:layout_width="0dp"
+    android:layout_width="156dp"
```

Comment:
Does setting the width mean we should remove the `layout_columnWeight` here?

Reply:
Yeah that makes sense